### ISSUE TITLE
[Broken Website] Instagram PWA on firefox nightly opens with light mode

### ISSUE THREAD

**Issue by Blaze-Thunder** [darkreader/darkreader#2382]
**Website**
www.instagram.com

**How to Reproduce**
> 1. Go to www.instagram.com site
> 2. Hover over homepage
> 3. See that when the PWA is opened, the website is white.
> 4. When I go to dark reader and turn the toggle off and on then it loads into dark mode

**Expected behavior**
When the instagram PWA is opened, it should load by default into dark mode.

**Actual behavior**
When instagram PWA is opened, it is loading in white mode.

**Screenshots**
![Screenshot_20200426-113903](https://user-images.githubusercontent.com/64011423/80299512-79f8c200-8784-11ea-9427-ad1f17d39005.png)

**System info:**
 - OS:Android
 - Browser Firefox Fenix Preview 
 - Darkreader Version 4.9.6

**Additional context**

**Comment by Gusted** [comment]
I assume that PWA means Progressive Web Application

**Comment by Blaze-Thunder** [comment]
@Gusted Yes right...

**Comment by Famiu** [comment]
I face the exact same issue, it would be great if this was fixed. One thing to note, however, is that the Instagram Splash Screen appears in Dark mode, but when the splash screen goes away, the website is awkwardly colored. Most of the colors are Light but in some places it's black, even though normally they're also white. So the cause might be the splash screen

**Comment by Blaze-Thunder** [comment]
Yes initial load has some awkward colors like white in some parts and black in some. But once I go into dark reader settings, disable and enable the site dark mode everything comes clearly in dark mode. No glitches.
The only problem is during the initial load of page.

**Comment by Famiu** [comment]
It seems like we both have the exact same issue, then

**Comment by Gusted** [comment]
Is their an option to open it without PWA?

**Comment by Blaze-Thunder** [comment]
I see that only option is to enable desktop site. But it is causing compatibility issues. Other than that if we just go to instagram.com it opens as PWA by default

**Comment by Gusted** [comment]
Hmm intresting behaviour of PWA

**Comment by Blaze-Thunder** [comment]
BTW Telegram PWA is working absolutely fine

**Comment by Gusted** [comment]
That's giving me some worries and seems like it's being instagram fault's if it's not PWA faults....

**Comment by Blaze-Thunder** [comment]
Hmm I see..

**Comment by CalvinWu4** [comment]
This is still an issue
